src/v/pandaproxy/schema_registry/types.h

```diff
@@ -395,4 +395,45 @@ from_string_view<compatibility_level>(std::string_view sv) {
       .default_match(std::nullopt);
 }
 
+enum class service_kind {
+    none = 0,
+    schema_registry,
+    http_proxy,
+};
+
+constexpr std::string_view to_string_view(service_kind kind) {
+    switch (kind) {
+    case service_kind::none:
+        return "none";
+    case service_kind::schema_registry:
+        return "schema-registry";
+    case service_kind::http_proxy:
+        return "http-proxy";
```

Comment:
Why does this exist inside schema registry?

Reply:
I was preparing for a future where we add a restart endpoint to the proxy. I'll move this.